cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -174,10 +174,8 @@ func main() {
 	cfg := configurator.NewConfigurator(configClient, stop, osmNamespace, osmMeshConfigName, msgBroker)
 
 	// Intitialize certificate manager/provider
-	certProviderConfig := providers.NewCertificateProviderConfig(kubeClient, kubeConfig, cfg, providers.Kind(certProviderKind), osmNamespace,
+	certManager, _, _, err := providers.GenerateCertificateManager(kubeClient, kubeConfig, cfg, providers.Kind(certProviderKind), osmNamespace,
```

Comment:
That's a good point! I ended up with the rename since the ProviderConfig went away, and theres' already a `certificate.NewCertManager` function.

I'd probably ere on the side of keeping it to remove that confusion, with the added caveat that this function is also going to go away completely in the near future.